assets/base.css

```
@@ -1099,10 +1097,6 @@ summary::-webkit-details-marker {
     width: calc(50% - 3.75rem / 2);
```

To avoid the weird bug you mentioned (https://screenshot.click/22-01-xpvxi-766mv.mp4), can we set the `grid--peek .grid__item` size to be based on the horizontal spacing?

I would probably suggest `calc(50% - var(--grid-mobile-horizontal-spacing) * 2)`.

```suggestion
    width: calc(50% - var(--grid-mobile-horizontal-spacing) * 2);
```